gradle/project.gradle.kts

```diff
@@ -247,7 +244,7 @@ tasks {
     }
 
     "run"(JavaExec::class) {
-        dependsOn(copyLibs, kotlinc, "jar")
```

Comment:
As I understand, this fixed incorrect task dependency and will improve performance. Right?

Reply:
I am not sure actually. But I don't see how this would improve performance. I don't even see why this would be useful at all.

Reply:
Update: It is necessary for widget compilation to actually work, since widgets depend on the code in the jar.

Shows us once again how not messing with things we do not understand is not a good idea. I fall for this trap again and again, even with my old code (from years ago).